### ISSUE TITLE
Primer should recommend validating editor, rather than post-facto validation

### ISSUE THREAD

**Issue by bdarcus** [citation-style-language/documentation#2]
Current primer says "After making changes to a CSL style, it is always a good idea to check whether the style still validates," which suggests it's a good idea to edit without real-time validation. 

You might insert something like this somewhere: "It is easiest to do XML editing with editing tools that support real-time completion and validation (such as emacs nxml mode)."

---
- Bitbucket: https://bitbucket.org/bdarcus/csl-docs/issue/8
- Originally Reported By: Bruce D'Arcus
- Originally Created At: 2011-02-15 18:53:42

**Comment by denismaier** [comment]
Maybe we should recommend atom instead of emacs?

**Comment by bdarcus** [comment]
> Maybe we should recommend atom instead of emacs?

Yes; see [the little package](https://github.com/bdarcus/atomic-csl) I created for atom.

**Comment by bwiernik** [comment]
@bdarcus Yeah, I think let's move this repo to CSL, and we can add a similar one for VS Code if a similar parent RNG validator is developed for that.
